style.css

```diff
@@ -179,38 +174,74 @@ ul {
   border: 1px solid #6070ff;
   border-radius: 8px;
   margin-left: 40px;
-  font-family: 'Poppins', sans-serif;
+}
+
+.about-section {
+  display: grid;
+  row-gap: 12px;
+  padding: 24px;
+  background-color: #fff;
+  border-top-right-radius: 80px;
 }
 
 .about-myself {
   color: #172b4d;
   font-size: 40px;
+  font-weight: 700;
 }
 
 .about {
   padding: 20px;
+  flex-direction: column;
+  font-weight: 400;
+  font-size: 16px;
+  color: #344563;
 }
 
-.text {
-  padding-left: 36px;
+.connect {
   font-weight: 500;
-  font-size: 20px;
-  color: #000;
-  width: 200px;
-  vertical-align: middle;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
 }
 
-.languages-frame {
+/* languages part */
+
+.row-1,
+.row-5,
+.row-6 {
   display: flex;
-  align-items: center;
-  margin-top: 48px;
-  margin-bottom: 20px;
+  justify-content: space-between;
+  font-weight: 500;
+  font-size: 20px;
 }
 
 .languages-list {
   display: flex;
   flex-direction: column;
-  gap: 12px;
+  margin-top: 16px;
+  gap: 24px;
+}
+
+.row-2,
+.row-3,
+.row-4 {
+  display: flex;
+  gap: 20px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+}
```

Comment:
- [ ] I would suggest you stick as much as possible to the Figma design's color, size, and font-family
### your design
![hsssssssssssssssssss](https://user-images.githubusercontent.com/81727988/178094116-e086fc1f-aa51-4d12-b4de-cdcd77ce4f3f.png)

### Figma design
![fffff](https://user-images.githubusercontent.com/81727988/178094123-91bc297f-6999-4d0c-a306-b358c3c7a362.png)